src/js/constants.js

```diff
@@ -46,12 +46,15 @@ export const MediaStateChangeEvents = {
   MEDIA_SEEKABLE: 'mediaseekablechange',
   MEDIA_PREVIEW_IMAGE: 'mediapreviewimagechange',
   MEDIA_PREVIEW_COORDS: 'mediapreviewcoordschange',
-  // MEDIA_CHROME_ATTRIBUTES: 'media-chrome-attributes',
-  // MEDIA_CONTROLLER: 'media-controller',
```

Comment:
does @cjpillsbury remember why these were commented out?

Reply:
If there was a good reason to keep them around it probably applied to the the other set of constants, and probably got carried over when creating multiple lists. These are attributes in a list events.

Reply:
they were only there for parity with `MediaUIAttributes`. I found it easier to A/B between the two with them there, but they can be removed.